net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/general.xml

```diff
@@ -5,4 +5,28 @@
         <type>checkbox</type>
         <help>This will activate the routing service.</help>
     </field>
+    <field>
+        <id>general.enablelogfile</id>
+        <label>create a logfile</label>
```

Comment:
I'd start the labels with upper char (for all the fields). I saw that in every menu "enable" starts low but everything else with upper char. Perhaps we streamline everything to "Enable"